setup.py

```diff
@@ -364,7 +364,7 @@ def get_dynamic_setup_params():
     license='GPLv3+',
     # Ansible will also make use of a system copy of python-six and
     # python-selectors2 if installed but use a Bundled copy if it's not.
-    python_requires='>=2.7,!=3.0.*,!=3.1.*,!=3.2.*,!=3.3.*,!=3.4.*',
+    python_requires='>=3.8',
```

Comment:
I've gone ahead and added this change now. I can remove it, we can keep it, we can change it later.  I'll discuss this with the other architects and confirm our plan.